modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -514,14 +514,25 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                     </xsd:element>
                     <xsd:element name="context-property" type="spring:entryType" minOccurs="0" maxOccurs="unbounded">
                         <xsd:annotation>
-                            <xsd:documentation>A property that wil be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
+                            <xsd:documentation>A property that will be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
                 </xsd:sequence>
+                <xsd:attribute name="timeBetweenTransformerEvictions" type="mule:substitutableInt">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                          Transformers are pooled for better throughput, since performing an XSL
```

Comment:
That's partially correct. Since the maxIdle attribute of the pool is equal to its max size, the objects will be created (and added to the pool) on demand, to support up until 32 concurrent consumers. After that, those objects will remain in the pool with no need to be rebuilt.

What this approach will do, it will create a new thread which has some unwanted implications (as per https://commons.apache.org/proper/commons-pool/api-1.6/org/apache/commons/pool/impl/GenericObjectPool.html) that will create the objects up to its max size, but will then keep running periodically.

A better solution IMO for having the prepopulation as you want is to add a initialTransformersCount property to XsltTransformer and update its doInitialise method to take that into account.